components/Trade/Advanced/TourSteps/index.tsx

```diff
@@ -256,10 +284,13 @@ export const tourConfig = [
         },
         // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
         action: () => {
-            // Enable the close order button
-            const closeOrderButton = document.getElementById('position-close-container') as HTMLButtonElement;
-            if (closeOrderButton) {
-                closeOrderButton.disabled = false;
+            const calculatorButton = document.getElementById('calculator-button') as HTMLElement;
+            if (calculatorButton) {
+                calculatorButton.style.zIndex = '0';
+            }
+            const depositButton = document.getElementById('deposit-button') as HTMLElement;
+            if (depositButton) {
+                depositButton.style.zIndex = '0';
```

Comment:
Why is this done every time?? I dont think this needs to be done every time, just after the deposit step